src/index.ts

```diff
@@ -90,8 +95,12 @@ export default class HDKeyring implements Keyring<SerializedHDKeyring> {
 
     this.#mnemonic = mnemonic
 
+    const passphrase = hdOptions.passphrase || ""
```

Comment:
```suggestion
    const passphrase = hdOptions.passphrase ?? ""
```

Let's stick to null coalescing here for clarity.

Reply:
This makes perfect sense, will do.